packages/gatsby-remark-prismjs/package.json

```
@@ -1,7 +1,7 @@
 {
-  "name": "gatsby-remark-prismjs",
+  "name": "@elken/gatsby-remark-prismjs",
   "description": "Adds syntax highlighting to code blocks at build time using PrismJS",
-  "version": "3.3.3",
+  "version": "0.3.3",
```

```suggestion
  "version": "3.3.3",
```

We don't need to change versions in a PR since versioning is handled by lerna